### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by ajgajg1134**
Just adding a simple contribution guide.

### CODE REVIEW

**Comment by liam-middlebrook** [comment]
I don't see any compelling reason to add this guide. There are already instructions in the document itself (articles.tex) that define the process. This only adds another location that would need to be changed if the process was altered. 

In addition why make a separate file from README.md? The README is short enough that anyone who is going to contribute should be able to read it with ease.

**Comment by ajgajg1134** [comment]
I disagree in that the document itself merely defines the process by which we ratify amendments. Wheras the purpose of this PR is to make it easier for those who wish to modify this repository which is how we currently store the constitution. I agree the README is really small too, I only separated it in case people wanted to expand on it further and therefore not clutter the README, but I see your point.

I'm curious what others have to say, specifically constitutional maintainers.

**Comment by JDrit** [comment]
I think that this would be useful l, especially for people who do not have an in depth knowledge of working with latex files. It does not have to be anything official so we can change the doc easily in the future. 

tl;dr I agree with Glaude

**Comment by gambogi** [comment]
I say we add to the current readme until it starts to feel unwieldy. Otherwise I do agree there is a need for a bit more documentation of how to build the constitution.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## Requirements
+Must have make, and pdflatex installed
+
+## Step by Step
+
+1. Fork this repo
+2. Make a new branch on your fork
+3. Modify the `.tex` files with your changes
+4. Run make *TWICE*
```
Comment (gambogi) [inline CONTRIBUTING.md]: Shouldn't have to run make twice anymore, the current makefile should compile twice so the links work. If that's not the case then we should make it the case.

**Comment by liam-middlebrook** [comment]
@ajgajg1134  I'm happy to merge this if you concatenate it all into README.md

**Comment by liam-middlebrook** [comment]
And squash these commits into one :smiley_cat: !

**Comment by ajgajg1134** [comment]
Squashed and moved to README.md
